src/models.py

```diff
@@ -28,9 +31,39 @@ def exists(username=None, id=None):
 			return User.select().where(User.username == username).exists()
 
 
-models = [User]
+class Directory(BaseModel):
+	id = IntegerField()
+	parent = IntegerField()
+	depth = IntegerField()
+	name = CharField()
+
+	def get_full_path(self):
+		if self.id is not None:
+			if self.id is not 0:
+				return Directory.get(Directory.id == self.parent).get_full_path() + '\\' + self.name
+			else:
+				return self.name
+
+#maybe what a File will look like
+class File(BaseModel):
+	id = IntegerField()
+	directory_id = IntegerField()
+	name = CharField()
+	content = BlobField()
+	extension = CharField()
+
+models = [User, Directory, File]
+
 for model in models: 
+	model.drop_table(fail_silently=True)
```

Comment:
This will work for now, but will delete all the data in any table that is dropped, so it'll need to be removed once we start wanting to persist data through deployments.

Reply:
I understand. It's just for messing with the default directories at will for now.